android-components/components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -254,16 +254,16 @@ class BrowserToolbar @JvmOverloads constructor(
 
     /**
      * Adds an action to be displayed on the right side of the toolbar (outside of the URL bounding
-     * box) in display mode.
+     * box) in display mode with an optional start margin.
      *
      * If there is not enough room to show all icons then some icons may be moved to an overflow
      * menu.
      *
      * Related:
      * https://developer.mozilla.org/en-US/Add-ons/WebExtensions/user_interface/Browser_action
      */
-    override fun addBrowserAction(action: Toolbar.Action) {
-        display.addBrowserAction(action)
+    override fun addBrowserAction(action: Toolbar.Action, marginStart: Int?) {
```

Comment:
does this signature also need `marginStart: Int? = null`?

Reply:
As this is an overriding function:
1.  will inherit the default value from the interface
2. it cannot specify its own default